iroha/iroha.py

```diff
@@ -26,14 +28,21 @@ class IrohaCrypto(object):
     """
 
     @staticmethod
-    def derive_public_key(private_key):
+    def derive_public_key(private_key, use_multihash=False, multihash_code=0x12):
```

Comment:
```suggestion
    def derive_public_key(private_key, use_multihash=False, multihash_code=0xed):
```

Reply:
fixed in my [patch](https://github.com/hyperledger/iroha-python/pull/54#issuecomment-642658788)